modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,121 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+
+[id='odo-sample-applications']
+= Sample applications
+include::openshift-developer-cli-attributes.adoc[]
+
+:context: odo-sample-applications
```

Comment:
Before all the content below is shown, we **NEED** to add a note that there Java and Node.JS are the only officially supported ones and to consult `odo catalog list components` before proceeding to check to see what is support and not supported.

Reply:
Note added.